services/rfq/relayer/cmd/commands.go

```diff
@@ -36,9 +37,14 @@ var runCommand = &cli.Command{
 		if err != nil {
 			return fmt.Errorf("could not read config file: %w", err)
 		}
-
 		metricsProvider := metrics.Get()
 
+		omniClient := omniClient.NewOmnirpcClient(cfg.OmniRPCURL, metricsProvider, omniClient.WithCaptureReqRes())
+
+		if err = cfg.ValidateTokenDecimals(c.Context, omniClient); err != nil {
```

Comment:
IMO it's a bit unnatural to do config validation in the cmd, my recommendation would be to call `validateTokenDecimals()` within the existing `Validate()` function. Can check `if omniClient == nil` so that we handle cases where the caller may not have access to a client